src/components/rhelGraphCard/rhelGraphCard.js

```diff
@@ -28,6 +32,15 @@ class RhelGraphCard extends React.Component {
       [rhelApiTypes.RHSM_API_QUERY_START_DATE]: startDate.toISOString(),
       [rhelApiTypes.RHSM_API_QUERY_END_DATE]: endDate.toISOString()
     });
+
+    setTimeout(() => {
+      this.setState({ width: this.containerRef.current.clientWidth });
+      window.addEventListener('resize', this.handleResize);
+    });
+  }
```

Comment:
question: not a fan of compensating for a component, outside of our control, to render with setTimeouts. this feels like something prone to breaking in the near future... 

this appears as an adaptation to some shortcoming of the component shouldn't this be handled by the chart component itself? Reason the CSS solution we had before was pretty straight forward, this just switched our CSS to a JS solution instead making it a little bit harder to follow

Reply:
+1 - it would be nice to not have to do this downstream.